.editorconfig

```diff
@@ -13,10 +13,10 @@ insert_final_newline = true
 max_line_length = off
 trim_trailing_whitespace = false
 
-[*.{json,yml,md,babelrc,remarkrc,prettierrc}]
+[*.py]
```

Comment:
Python files? In this repo?

Reply:
Yeah, let me take that out.  it's my default one